combat-trainer.lic

```diff
@@ -873,7 +878,10 @@ class LootProcess
   end
 
   def check_skinning(mob_noun, game_state)
+    already_arranged = false
     if @dissect && game_state.dissectable?(mob_noun)
+      arrange_mob(mob_noun, game_state) if @arrange_for_dissect
```

Comment:
Should the conditional use `@always_arrange`? Or should line 390 be named `@arrange_for_dissect`?

Reply:
I left the old name in. Sorry about that, and thank you!